### ISSUE TITLE
Customize Installation Path Format

### ISSUE THREAD

**Issue by aaronsky** [RobotsAndPencils/xcodes#197]
Hey folks, thank you for everything you've done in making this and Xcodes.app the successes they are. They're really great tools and I appreciate the work you've done. 🙏🏼 

I've got an enhancement request, possibly unreasonable, that would make xcodes fit better into my weird little workflow. Unlike many others, I don't like to rename Xcode.app from whatever name it's been extracted as. So an installation strategy where Xcode.app is renamed to `/Applications/Xcode-13.3.1.app` won't fly. What I do, which I've never seen anyone else do, is organize builds by their revision string, meaning Xcode 13.3.1 would be installed to `/Applications/Xcode/13E500a/Xcode.app`. 

I won't pretend I have a great reason to do this, I'm just superstitious of renaming Xcode, and I often have a lot of Xcode versions installed. I can also imagine that there are others out there who would benefit from more flexibility in the way xcodes installs versions beyond this specific style, besides the included `XCODES_DIRECTORY` option. Does this seem like something that would align with the goals of the project?

**Comment by kidbrax** [comment]
it would be great to have something like an --output switch where we can specify what we want the resulting .app to be named

**Comment by neoswallow** [comment]
I second this option as Azure DevOps Pipelines and GitHub Actions macOS runners use Xcode_{version string}.app format.
So in order to setup a self-hosted agent, resulting Xcode name needs to be renamed.

**Comment by zoltan-magyar** [comment]
I have have the guts to bring up my suggestion as well here, since I understand that it very well be out of scope for the project.

I generally don't have multiple Xcode installs (I know I know, that's the whole point.)

The reason I use xcodes is simply because it's still the best way to install Xcode, considering how terrible the AppStore is and what a chore it is to download the installer and check for updates otherwise.

Basically, what I would love to see is a "single" Xcode mode, where Xcode.app is the final name of the extracted application that I can update seamlessly with the CLI.

Like I said, I know this is very clearly not what the CLI is about, but considering the fact that there is really no tool that could do this, it would be a great addition in my opinion.

**Comment by acevif** [comment]
@neoswallow I encountered the same issue.

Creating a symlink on the self-hosted runner using the following command worked for me:

```
ln -s /Applications/Xcode-15.4.0.app/ /Applications/Xcode_15.4.app
```

I hope this helps!
